src/ROUTES.ts

```diff
@@ -37,16 +36,9 @@ const ROUTES = {
     ALL_SETTINGS: 'all-settings',
 
     SEARCH_CENTRAL_PANE: {
-        route: '/search/:query',
-        getRoute: (searchQuery: SearchQuery, queryParams?: AuthScreensParamList['Search_Central_Pane']) => {
-            const {sortBy, sortOrder} = queryParams ?? {};
-
-            if (!sortBy && !sortOrder) {
-                return `search/${searchQuery}` as const;
-            }
-
-            return `search/${searchQuery}?sortBy=${sortBy}&sortOrder=${sortOrder}` as const;
-        },
+        route: 'search',
+        getRoute: ({query, queryKind = CONST.SEARCH.QUERY_KIND.CANNED_QUERY, policyIDs}: {query: SearchQueryString; queryKind?: QueryKind; policyIDs?: string}) =>
+            `search?${queryKind === CONST.SEARCH.QUERY_KIND.CANNED_QUERY ? 'q' : 'cq'}=${query}${policyIDs ? `&policyIDs=${policyIDs}` : ''}` as const,
```

Comment:
I think this is a transitional step.

BTW will current grammar handle `policyID`? I don't see this keyword in the `.peggy` file

Reply:
Ah makes sense that we do it like this for now. We should eventually move it to the AST and include it in the peggy grammar too

Reply:
Do we need a separate issue to migrate the `policyID` param to the AST?

Reply:
Do you think it should be in the root of `QueryJSON`? So something like 
```ts
type JSONQuery = {
    input: string;
    hash: number;
    type: string;
    status: string;
    sortBy: string;
    sortOrder: string;
    offset: number;
    filters: ASTNode;
    policyID: string;
};
```

Also currently it is called policyIDs, as the assumption was that we wanted to be able to use multiple policyIDs in the future. Is this still the case?

Reply:
We'll add it to filters as `policyID`. So the AST will look like this:

```
{filters: {
    "operator": "eq",
    "left": "policyID",
    "right": "<policyID>"
```

I think we should use `policyID` singular from now on since we have a bunch of other filters, e.g. `category`, `tag`, etc that accepts multiple values but are all singular too.

Reply:
I hope I understand this correctly. I will add `policyID` to grammar and keep it inside AST, while removing it from URL

Reply:
read my comment here: https://github.com/Expensify/App/pull/45617#issuecomment-2250322113